InvenTree/company/api.py

```diff
@@ -313,6 +313,12 @@ def filter_queryset(self, queryset):
             active = str2bool(active)
             queryset = queryset.filter(part__active=active)
 
+        # Order by last_updated?
```

Comment:
Ordering should not be handled in the `filter_queryset` method. You should add "last_updated" to the `ordering_fields` list (search for "ordering_fields" in `api.py`)